src/plugins/ResizeReorder/ResizerKnob.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import cx from 'cx';
+import joinClasses from 'joinClasses';
+import FixedDataTableEventHelper from 'FixedDataTableEventHelper';
+import ResizerLine from './ResizerLine';
+import emptyFunction from 'emptyFunction';
+
+
+
+class ResizerKnob extends React.Component {
+  initialState = {
+    isColumnResizing: undefined,
+    left: undefined,
+    width: undefined,
+    minWidth: undefined,
+    maxWidth: undefined,
+    initialEvent: undefined,
+    key: undefined
+  };
+
+  state = {...this.initialState};
+
+  clearState = () => {
+    this.setState(
+      this.initialState
+    );
+  }
+
+  componentDidMount(){
+    this.setState({
+      instance: this.curRef
+    });
+  }
+
+  render() {
+    const dragKnob =
+    <ResizerLine
+      height={this.props.resizerLineHeight}
+      initialWidth={this.state.width || 0}
```

Comment:
if the initial state is zero, then would you need to shortcircuit with 0 ? 
Also same for below props.

Reply:
I am not sure, I took this part from [here](https://github.com/schrodinger/fixed-data-table-2/blob/a1c62989fca650a39cb24df7844aca06d9b9f396/src/FixedDataTable.js#L801).